compiler/modules/front.syntax/astString.bal

```diff
@@ -315,9 +313,35 @@ function typeDescToWords(Word[] w, TypeDesc td, boolean|BinaryTypeOp wrap = fals
     else if td is SingletonTypeDesc {
         valueToWords(w, td.value);
     }
+    else if td is UnaryTypeDesc {
+        w.push(td.op, CLING);
+        typeDescToWords(w, td.td);
+    }
+    else if td is XmlSequenceTypeDesc {
+        w.push("xml<", CLING);
```

Comment:
Each token should be pushed separately, as elsewhere in this file i.e. `"xml", "<"`

Reply:
Fixed.